e2e/__tests__/jasmineAsync.test.ts

```diff
@@ -115,6 +115,14 @@ describe('async jasmine', () => {
     expect(json.testResults[0].message).toMatch(/concurrent test fails/);
   });
 
+  it('works with concurrent.each', () => {
```

Comment:
@SimenB @mattphillips Not sure why this test is not passing.
The test I've written [directly](https://github.com/M4rk9696/jest/blob/concurrent-each/packages/jest-each/src/__tests__/index.test.ts) without execa seem to pass

Reply:
Not seen these tests before but shouldn't all of the expected values be doubled (given that there are two tests with two rows)?

Reply:
I had that confusion where it will be doubled, but I seem to be getting that zero tests have been run

Reply:
This seems to be the behaviour of `each`.

Reply:
Called `each` incorrectly, closed now